compiler/src/iree/compiler/Dialect/Stream/Transforms/EncodeTensors.cpp

```diff
@@ -478,7 +514,25 @@ struct EncodeTensorFillOp
     if (!pattern) {
       return rewriter.notifyMatchFailure(
           op, "unsupported pattern width; encoding policy required");
-    } else if (pattern.getType().getIntOrFloatBitWidth() > 32) {
+    }
+    unsigned bitWidth = 0;
+    if (isa<ComplexType>(pattern.getType())) {
+      bitWidth = dyn_cast<ComplexType>(pattern.getType())
+                     .getElementType()
+                     .getIntOrFloatBitWidth();
+    } else {
+      bitWidth = pattern.getType().getIntOrFloatBitWidth();
+    }
+    if (bitWidth > 64) {
+      // This happens mostly when complex<f64> is used as a input type for
+      // splat. complex<type> is broken down into a 2xtype value with the real
+      // field occupying the sizeof(type) MSB bits and the imaginary field
+      // occupying the rest. At this moment, splats with size > 64 is not
+      // implemented so we error out here.
+      return rewriter.notifyMatchFailure(
+          op, "unsupported bitWith; encoding policy required");
```

Comment:
type for bitWith -> bitwidth

We should also specify that bitwidth is greater than 64-bit in the error message.

Reply:
Fixed both.